modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -69,7 +69,7 @@ def handle_210966_authenticated
       end
 
       def submit_form_to_central_mail
-        parsed_form_data = JSON.parse(params.to_json)
+        parsed_form_data = form_is210966 ? handle_210966_data : JSON.parse(params.to_json)
```

Comment:
Just curious, why are no other forms doing this (why are there no other if conditions in this area)? surely other forms need to do transformations. I would expect this to live along side similar conditionals? otherwise seems fine.

Reply:
This is a great question and one that I considered for a bit. There are indeed many more small string manipulations happening in the `erb` template files themselves. So far they haven't _required_ a data structure like the `roles` hash I introduced here but I think it's worth maybe putting these into the model files for each form. I tend to take a wait-and-see approach in times like this though, where we only have ~~one~~two such examples (see bottom of this comment) where that really makes sense. But a moderate refactor may be in order if I see this type of thing again.

This line might change to something like:
```
SimpleFormsApi::VBA210845.new(params).parsed_form_data
```
and let the model handle these things.

The closest thing I've done along these lines before would be this:
```
  "F[0].#subform[1].RadioButtonList[0]": "<%= {'SPOUSE' => 1, 'PARENT' => 2, 'CHILD' => 3}[data['claimant_identification']] %>",
```
It would be nice to pull this out into a model method.